src/screens/app/screens/Dashboard/components/TaskAddedModal/TaskAddedModal.context.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, createContext, useMemo, useState } from 'react'
+
+interface TaskAdded {
+  emoji: string
+  points: number
+}
+
+interface TaskAddedContextProps {
+  openTaskAddedModal: (task: Task, taskDifficultyIndex: number) => void
+  closeTaskAddedModal: () => void
+  taskAdded: TaskAdded
+}
+
+// @ts-ignore
+const TaskAddedModalContext = createContext<TaskAddedContextProps>({})
+
+const TaskAddedModalContextProvider: FC = ({ children }) => {
+  const [taskAdded, setTaskAdded] = useState<TaskAdded | null>(null)
+
+  const openTaskAddedModal: TaskAddedContextProps['openTaskAddedModal'] = (
```

Comment:
TaskAddedContextProps.openTaskAddedModal ? is it a type ? use Uppercase ?

Reply:
I don't understand :) 
L8 we have the definition of the context with all methods available. Here I force the type just to be sure to always have the good definition.